Tasks/LambdaDeployFunction/TaskOperations.ts

```diff
@@ -167,6 +171,10 @@ export class TaskOperations {
         try {
             const response = await this.lambdaClient.createFunction(request).promise()
 
+            if (!response.FunctionArn) {
```

Comment:
An empty/undefined `response.FunctionArn` value doesn't seem expected. Is there a reason we wouldn't throw in this case instead?

Reply:
ditto

Reply:
Just checking if you were going to add the throw -- I don't see it in the current changes

Reply:
Oops, it helps to save the file after I make changes. Fixed.